tools/StaticAnalysis/Exceptions/Az.StorageSync/SignatureIssues.csv

```diff
@@ -1,4 +1,12 @@
 "AssemblyFileName","ClassName","Target","Severity","ProblemId","Description","Remediation"
 "Microsoft.Azure.PowerShell.Cmdlets.StorageSync.dll","Microsoft.Azure.Commands.StorageSync.Evaluation.Cmdlets.InvokeCompatibilityCheckCmdlet","Invoke-AzStorageSyncCompatibilityCheck","1","8100","Invoke-AzStorageSyncCompatibilityCheck Does not support ShouldProcess but the cmdlet verb Invoke indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
 "Microsoft.Azure.PowerShell.Cmdlets.StorageSync.dll","Microsoft.Azure.Commands.StorageSync.Evaluation.Cmdlets.InvokeCompatibilityCheckCmdlet","Invoke-AzStorageSyncCompatibilityCheck","1","8410","Parameter SkipNamespaceChecks of cmdlet Invoke-AzStorageSyncCompatibilityCheck does not follow the enforced naming convention of using a singular noun for a parameter name.","Consider using a singular noun for the parameter name."
```

Comment:
Note that your management client shoudl be published to Nuget, not included here, befroe the PR can be merged.